crates/typst/src/layout/align.rs

```diff
@@ -14,14 +14,56 @@ use crate::text::TextElem;
 /// Aligns content horizontally and vertically.
 ///
 /// # Example
+/// Let's start with centering our content:
```

Comment:
Maybe more consistent with the suggestion right below
```suggestion
/// Let's start with centering our content horizontally:
```